_includes/head.html

```diff
@@ -17,5 +17,5 @@
 
 <!-- For all browsers -->
 <link rel="stylesheet" href="{{ base_path }}/assets/css/main.css">
-
-<meta http-equiv="cleartype" content="on">
\ No newline at end of file
+<link rel="stylesheet" href="{{ base_path }}/assets/css/primer.css"
```

Comment:
Thanks for the 👀 @stoe! After I opened this I actually found a way to include it via the Jekyll config, so I plan to give that a shot.